public/manifest.json

```diff
@@ -1,25 +1,27 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Som Timer",
```

Comment:
Great job on the manifest, team. This already looks great on mobile and will look even better once this manifest is deployed!